### PR TITLE
Update iris docs

### PR DESCRIPTION
**Pull request by tinnus-napbus**
Iris docs again

Have fixed headers and links for the new framework

@drbeefsupreme

### CODE REVIEW

File: content/docs/arvo/iris/data-types.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
++++
+title = "Data Types"
+weight = 2
+template = "doc.html"
++++
+
+Iris itself only has three structures defined in `lull.hoon`, but it also uses [$http](/docs/arvo/eyre/data-types#http) structures which are shared between Iris and Eyre. The `$http` structures are detailed in the [Eyre](/docs/arvo/eyre/data-types) Data Types documentation.
```
Comment (drbeefsupreme) [inline content/docs/arvo/iris/data-types.md]:
```suggestion
Iris itself only has three structures defined in `lull.hoon`, but it also uses `$http` structures which are shared between Iris and Eyre. The `$http` structures are detailed in the [Eyre](/docs/arvo/eyre/data-types#http) Data Types documentation.
```

File: content/docs/arvo/iris/tasks.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
++++
+title = "API Reference"
+weight = 3
+template = "doc.html"
++++
+
+This document details the `task`s that can be sent to Iris. Iris only has a couple of `task`s you can use: [%request](#request) and [%cancel-request](#cancel-request).
```
Comment (drbeefsupreme) [inline content/docs/arvo/iris/tasks.md]:
```suggestion
This document details the `task`s that can be sent to Iris. Iris only has a few `task`s besides the standard vane tasks: [%request](#request), [%cancel-request](#cancel-request), and [%receive](#receive).
```
Feel free to ignore this suggestion. I think you meant that there were only two tasks that you'd use from within Arvo, but my first reading made me think there were only two tasks at all, so I was then surprised when I saw a third one.

**Review by drbeefsupreme — CHANGES_REQUESTED** [review]
LGTM for style besides my two short comments. I believe Iris is @joemfb 's domain so he should check it for technical accuracy.

**Comment by drbeefsupreme** [comment]
Maybe @Fang- if @joemfb doesn't have the bandwidth

File: content/docs/arvo/iris/tasks.md (hunk 2 above)
Comment (tinnus-napbus) [reply content/docs/arvo/iris/tasks.md]: I have rephrased this in https://github.com/urbit/urbit.org/pull/730/commits/4268a4ff7363cb065b4c0dece330dbf20e13ae7b

File: content/docs/arvo/iris/iris.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
++++
+title = "Overview"
+weight = 1
+template = "doc.html"
++++
+
+Iris is the HTTP client vane.
+
+Gall agents and Spider threads pass Iris HTTP requests, Iris makes the request to the specified URL, and finally returns the HTTP response back to the app or thread that requested it.
```
Comment (Fang-) [inline content/docs/arvo/iris/iris.md]: Similar to what I (iirc) mentioned in one of your other PRs, technically any part of the system can send Iris tasks, but they're just most common in userspace.

File: content/docs/arvo/iris/tasks.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+Iris does not return any `gift` in response to a `%cancel-request` `task`. You will also not receive any `gift` back from the original `%request` `task` you've cancelled.
+
+## `%receive`
+
+```hoon
+[%receive id=@ud =http-event:http]
+```
+
+Receives HTTP data from outside. This `task` is sent to Iris by the runtime, you would not used it manually.
```
Comment (Fang-) [inline content/docs/arvo/iris/tasks.md]:
```suggestion
Receives HTTP data from outside. This `task` is sent to Iris by the runtime, you would not use it manually.
```

**Review by Fang- — APPROVED** [review]
This looks good! Only the tiniest of nits below.

**Comment by tinnus-napbus** [comment]
cool I've reworded the iris introduction so it doesn't specifically mention agents or threads and fixed typo per @Fang- comments

**Pull request merged**